spec/solidus/gateway/braintree_gateway_spec.rb

```diff
@@ -348,6 +348,8 @@
             it 'sends a bill address' do
               expected_params = {
                 billing: {
+                  first_name: address.first_name,
+                  last_name: address.last_name,
```

Comment:
Do we need an assertion that the first and last name from the address is send to braintree?